src/robot_dart/sensor/sensor.cpp

```diff
@@ -43,8 +43,18 @@ namespace robot_dart {
         void Sensor::set_pose(const Eigen::Isometry3d& tf) { _world_pose = tf; }
         const Eigen::Isometry3d& Sensor::pose() const { return _world_pose; }
 
+        void Sensor::detach() {
+            _attached_to_body = false;
+            _attached_to_joint = false;
+            _body_attached = NULL;
```

Comment:
Can you use `nullptr` instead of `NULL`?

Reply:
sure, old habits :)